client/src/components/Profile/AboutCard.tsx

```diff
@@ -1,63 +1,59 @@
-import * as React from 'react';
-import isEqual from 'lodash/isEqual';
 import { Typography, Input } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { ChangeEvent, memo, useState } from 'react';
 import CommonCard from './CommonCard';
-import { GeneralInfo } from 'common/models/profile';
-import { ConfigurableProfilePermissions } from 'common/models/profile';
-import { ChangedPermissionsSettings } from 'pages/profile';
-import { CheckboxChangeEvent } from 'antd/lib/checkbox';
 
 const { Paragraph, Text } = Typography;
 const { TextArea } = Input;
 
-import { InfoCircleOutlined } from '@ant-design/icons';
-
 type Props = {
-  data: GeneralInfo;
+  data: string;
   isEditingModeEnabled: boolean;
-  permissionsSettings?: ConfigurableProfilePermissions;
-  onPermissionsSettingsChange: (event: CheckboxChangeEvent, settings: ChangedPermissionsSettings) => void;
   onProfileSettingsChange: (event: any, path: string) => void;
+  saveProfile: () => void;
+};
+
+const AboutCard = ({ isEditingModeEnabled, onProfileSettingsChange, saveProfile, data: aboutMyself }: Props) => {
```

Comment:
Instead of renaming variable by destructuring assignment you can just use different name of property

Reply:
I'll stick with card prop data, so all profile cards would have the same data prop name. Removed renaming destructuring.